client/styles/colours.ts

```diff
@@ -0,0 +1,38 @@
+// Primary colors
```

Comment:
If we use `colours` as the name of the object, then we should go all out 🚀 

```suggestion
// Primary colours
```

Reply:
oops good catch!